src/payment/data/service.js

```diff
@@ -3,12 +3,68 @@ import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 
 import handleRequestError from './handleRequestError';
 import { transformResults } from './utils';
+import { isWaffleFlagEnabled } from '../../data/waffleFlags';
+import { WAFFLE_FLAGS } from './constants';
 
 ensureConfig([
   'ECOMMERCE_BASE_URL',
   'LMS_BASE_URL',
 ], 'payment API service');
 
+/* eslint-disable no-use-before-define */ /* Chicken vs Egg...
+    but it is preferred the interpreter pickup errors rather than use magic strings */
+
+/**
+ * Ecommerce IDA/Coordinator IDA URLs keyed by the function name they are meant for.
+ *
+ * @type {{[p: string]: string}}
+ */
+const urlsByFunction = {
+  /* Shared common paths */
+  [getClientSecret.name]: '/bff/payment/v0/capture-context', // no forward slash at the end or we will explode
```

Comment:
"We will explode" is colorful, but not really accurate. Could we say that ecommerce will 404?

Reply:
i dont recall the exact error, but im coonfusing whether its BAD_METHOD or FILE_NOT_FOUND. But i will clarify.